contracts/contracts/RaidenMicroTransferChannels.sol

```diff
@@ -88,10 +95,22 @@ contract RaidenMicroTransferChannels {
         challenge_period = _challenge_period;
     }
 
+    /// @dev Sets the address for the latest contract version
+    /// @param _latest_version_address The address for the latest contract version.
+    function setLatestVersionAddress(address _latest_version_address) public isOwner {
+        require(addressHasCode(_latest_version_address));
```

Comment:
not sure if this check has to be here, I could use a precalculated address here, which I want to deploy the contract to _after_ I changed all "references"

Reply:
I didn't understand how this works, especially the "after I changed all "references"".
So, you precalculate the new contract address and use it for deployment. Then you still have to change `_latest_version_address` for the old contracts with this setter.

Reply:
Yes, that's correct, doesn't make sense this way. I thought of a scenario where it's crucial to set the address before I deploy, but there is no such scenario.